testsuite/preview/expansion/src/test/java/org/wildfly/test/integration/observability/opentelemetry/LoggingServerSetupTask.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.integration.observability.opentelemetry;
+
+import org.jboss.as.test.shared.ManagementServerSetupTask;
+
+/**
+ * A ServerSetupTask to set up logger categories to file handlers for log messages check in the tests.
+ *
+ * @author <a href="mailto:aoingl@gmail.com">Lin Gao</a>
+ */
+public class LoggingServerSetupTask extends ManagementServerSetupTask {
+
+    public LoggingServerSetupTask() {
+        this(SMALLRYE_OPENTELEMETRY_LOG_FILE, VERTX_FEATURE_PACK_LOG_FILE);
+    }
+
+    static final String SMALLRYE_OPENTELEMETRY_LOG_FILE = "smallrye-opentelemetry.log";
+    static final String VERTX_FEATURE_PACK_LOG_FILE = "vertx-feature-pack.log";
+
+    public LoggingServerSetupTask(String smallryeOpentelemetryLogFile, String vertxSubsystemLogFile) {
+        super(createContainerConfigurationBuilder()
+                .setupScript(createScriptBuilder()
+                        .startBatch()
+                        .add("/subsystem=logging/file-handler=otel-exporter:add(file={relative-to=jboss.server.log.dir, path=" + smallryeOpentelemetryLogFile + "}, append=false, level=DEBUG)")
```

Comment:
FWIW the preferred way is to use the method variant with a pattern and %s expression with params, i.e. this one `org.jboss.as.test.shared.ManagementServerSetupTask.CommandSet#add(java.lang.String, java.lang.Object...)` but this works too...

Reply:
I am glad to know that, thank you :)